src/docs/product/performance/filters-display.mdx

```diff
@@ -69,4 +69,6 @@ The [Display Filter](#display-filter) controls what displays in the table column
 - Unique Users
 - [User Misery](/product/performance/metrics/#user-misery)
 
-If you have transactions you frequently return to, you can mark up to 10 transactions as a key transaction by clicking the star in the corresponding row. Clicking the star ensures that these key transactions are always surfaced to the top of the list. You can also mark a transaction as a key transaction in its corresponding [summary view](/product/performance/transaction-summary/#key-transaction).
+### Key Transactions
+
```

Comment:
```suggestion
<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony. If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>

```